en/docs/develop/mi-for-vscode/install-wso2-mi-for-vscode.md

```diff
@@ -1,22 +1,16 @@
 # Install Micro Integrator for VS Code
 
-This page walks you through how to get started with the Micro Integrator for VS Code extension by installing it.
+This page walks you through how to get started with the Micro Integrator for VS Code extension.
 
 ## Installation Prerequisites
```

Comment:
Let's use sentence case for subheadings except for proper nouns, acronyms, and product names.